optionB/src/components/floorPlan/model.xml

```diff
@@ -0,0 +1,61 @@
+<model>
```

Comment:
I'm going to upload a new `xml` model. Perhaps we can `import` this from the root of the project so that its not in more than one place and also is linked to the latest `model.xml`?

Reply:
Create a react app isn't able to import from outside of the src/ directory, but I can replace the file here with the latest.